pkg/apiaddresses/controller.go

```diff
@@ -26,7 +26,10 @@ func Register(ctx context.Context, runtime *config.ControlRuntime, endpoints con
 
 	cl, err := etcd.GetClient(h.ctx, h.runtime, "https://127.0.0.1:2379")
 	if err != nil {
-		return err
+		cl, err = etcd.GetClient(h.ctx, h.runtime, "https://[::1]:2379")
```

Comment:
You can actually give Get client multiple endpoint addresses and grpc should try them all - does it work if you just pass them both into the first Get client?

You could also try removing the address entirely so that it falls back to the default, which should work fine. I think I just missed removing this explicit specification of the default address.